src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -288,8 +288,6 @@ private static string GetItemKind(DeclaredSymbolInfo declaredSymbolInfo)
             {
                 case DeclaredSymbolInfoKind.Class:
                     return NavigateToItemKind.Class;
-                case DeclaredSymbolInfoKind.Record:
-                    return NavigateToItemKind.Record;
```

Comment:
i'm a little confused how this works.  if you remove this entry, and we do run into a record, won't we crash in this switch?